analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ClassShouldNotBeEmpty.CSharp9.cs

```diff
@@ -1,44 +1,61 @@
 ﻿using System;
 
-record EmptyRecord1();                                      // Noncompliant {{Remove this empty record, write its code or make it an "interface".}}
-//     ^^^^^^^^^^^^
-record EmptyRecord2() { };                                  // Noncompliant
+namespace Compliant
```

Comment:
It's fine to reorganize the test cases by grouping the code using namespaces, but
- if you do it in one of the test files then do it for the others as well for S2094, for the sake of consistency
- next time please do it in a separate commit. That will make it easier for us to review the PR, because we'd see what are the places where only the indentation has changed and what is completely new.

Reply:
I did not want to put effort in rewriting all yet, as I wanted agreement first. ;)

Sorry for not putting it in a different commit.

Reply:
@zsolt-kolbay-sonarsource I think I addressed all issues you raised?

Reply:
Apologies. Didn't see the notification.

Reply:
/azp run [Sonar.Net](http://sonar.net/)